pkg/k8sutils/connect.go

```diff
@@ -0,0 +1,38 @@
+package k8sutils
+
+import (
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	keptnutils "github.com/keptn/kubernetes-utils/pkg"
+	"k8s.io/client-go/kubernetes"
+	"keptn-sandbox/job-executor-service/pkg/config"
+)
+
+// k8sImpl is used to interact with kubernetes jobs
+type k8sImpl struct {
```

Comment:
Maybe for a future improvement: We could store the `*kubernetes.Clientset` in `k8sImpl`. This would result in shorter methods in `K8s`.

Reply:
Yes, some of the method signatures are getting quite long